File: sample/src/main/java/androidx2j/sample/MainActivity.java
package androidx2j.sample;

import android.app.Activity;
import android.os.Bundle;
import android.view.LayoutInflater;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        LayoutInflater.from(this).inflate(R.layout.fragmetn_layout, null);
        LayoutInflater.from(this).inflate(R.layout.fragmetn_layout, null, false);
    }

}

File: sample-app/src/main/java/androidx2j/sample/app/MainActivity.kt
package androidx2j.sample.app

import android.app.Activity
import android.os.Bundle
import android.view.LayoutInflater

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        LayoutInflater.from(this).inflate(R.layout.fragment_layout, null)
        LayoutInflater.from(this).inflate(R.layout.fragment_layout, null, false)
    }
}